src/main/java/nextstep/courses/domain/course/NowBatchNo.java

```diff
@@ -0,0 +1,19 @@
+package nextstep.courses.domain.course;
+
+public class NowBatchNo {
+
+  private int nowBatchNo;
+
+  public NowBatchNo(int nowBatchNo) {
+    this.nowBatchNo = nowBatchNo;
+  }
+
+  public int createdBatch() {
+    ++nowBatchNo;
+    return nowBatchNo;
+  }
```

Comment:
```suggestion
  public NowBatchNo createdBatch() {
    return new NowBatchNo(++nowBatchNo);
  }
```

immutable로 구현하는 연습을 해보는 것은 어떨까?